sdks/python/apache_beam/runners/direct/direct_runner.py

```diff
@@ -90,15 +92,14 @@ class DirectRunner(PipelineRunner):
   """Executes a single pipeline on the local machine."""
 
   # A list of PTransformOverride objects to be applied before running a pipeline
-  # using DirectRunner.
-  # Currently this only works for overrides where the input and output types do
-  # not change.
-  # For internal SDK use only. This should not be updated by Beam pipeline
-  # authors.
```

Comment:
<!--new_thread; commit:022e74460433e0ad9b44356229151c0d7b0af351; resolved:0-->
Why the change here?